### PR TITLE
NTBS-3071: Make test result summaries consistent

### PR DESCRIPTION
**Pull request by LeoCie**
Made the ranking a lookup table, as it is used in a few different places. 
Made the logic for all the different test result summaries consistent (with how we do it for the `Initial...` fields)

### CODE REVIEW

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql [hunk 1]
```diff
@@ -411,7 +379,6 @@ BEGIN TRY
 	WHERE rr.SourceSystem = 'NTBS'
 
 	DROP TABLE #TempDiseaseSites
-	DROP TABLE #TempManualTestResult
 	DROP TABLE #TempSocialContextVenues
 
 	--'Sample taken' should be set if there are any manually-entered test results which are NOT of type chest x-ray
```
Comment (LeoCie) [inline source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql:384]: I've spotted this comment too, should we be also removing Chest CT results when looking for SampleTaken?

Comment (JoshCadney) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql:384]: I assume that we should, but I'm not certain. @adilmirzaphe What do you think?

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql (hunk 1 above)
Comment (adilmirzaphe) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql:384]: @JoshCadney  vaguely recall this being discussed with Esther. Might be worth clarifying in catch up meeting tomorrow.

Comment (LeoCie) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql:384]: I'm going to put it in this ticket in JIRA, as it's a small change and still makes sense in this ticket.

**Pull request merged**